ui/radar/radar.js

```diff
@@ -120,7 +125,12 @@ class Radar {
     // option overwrite
     if (monster.rank in options.RankOptions)
       options = Object.assign({}, this.options, options.RankOptions[monster.rank]);
+    // Chceck if monster rank enabled or not in config
+    if(monster.rank === "B" && options.BRankEnabled === false)
+      return;
     if (options.Type === 'mob') {
+      if (options.Enabled === false)
```

Comment:
This can just be `!options.Enabled`.  Also, I think this check should be above the `if (options.Type === 'mob') {` line so it could apply to non-mobs if for some reason somebody has those defined.